src/schedule/dma_single_chan_domain.c

```diff
@@ -224,7 +224,7 @@ static int dma_single_chan_domain_register(struct ll_schedule_domain *domain,
 		register_needed = false;
 	}
 
-	tr_info(&ll_tr, "dma_single_chan_domain_register(): registering on channel with period %u",
+	tr_info(&ll_tr, "dma_single_chan_domain_register(): registering on channel with period %llu",
```

Comment:
Do we have a better trace system that works with %llu properly?

Reply:
Zephyr does, but I've a few of these trace changes that I may push back into a trace string PR. Problem today is that both Zephyr and SOF are not fully supporting all the libc printf formatters. This is a WIP....

Reply:
I have had issues with %llu/%llx/... when testing my interrupt controller drivers last year, that's why I'm asking. I think parameters are passed weirdly and extracted _even more_ weirdly which will cause incompatibilities if the datatype isn't 32 bits on-stack (char, short and int are all 32-bit due to how variadic functions work)